_posts/2018-07-18-inception-pipelines-pt5.markdown

```diff
@@ -0,0 +1,84 @@
+
+* Keep the concerns of each separate (what the permissions are vs who has them)
+* Allows reuse of the policies if desired.
+* Policies can be reused internally without the cross-account assumption clause being included.
+
+![iam users groups roles]({{ site.url }}/img/inception-pipelines/part-5-users-groups-roles.png)
+
+### Child Account
+
+The child account owns what developers can perform inside of it, which is does by defining IAM roles that can be assumed by the root account. In the example code, it does this by this snippet in child/aws_capability_iam.yml:
+
+```yaml
+Statement:
+  - Action: ['sts:AssumeRole']
+    Effect: 'Allow'
+    Principal:
+      AWS:
+        - !Sub 'arn:aws:iam::${ParamRootAccount}:root'
+```
+
+Also in the same file is the DeveloperPolicy document which outlines what the developer can do. In the example code, the developer can manage the Inception Pipeline, view logs, read from S3 buckets and that's about it.
+
+### Root Account
+
+The root account is the central location for managing IAM users and what they have access to. The file root/aws_capability_iam.yml defines an IAM Group (with matching IAM Policy) that can assume the DeveloperRole in the child account. Depending on your use case (number of accounts, developers, etc) you could just define a single group that has access to all your child accounts. or define a Group-per-Account. It's up to you.
+
+The final file is root/aws_capability_iam_users.yml. In this file you create your IAM User and then assign them to all the Group(s) that they need. If the developer leaves the company, simply delete the IAM User and any group assignments and redeploy. Within a minute they will no longer have any access. Alternatively, you can just remove them from the Group(s) if it's a temporary change e.g. long service leave.
+
+## Where Do I Get The Seed Files
+
+Two complete, working pipelines are available here on GitHub, one pipeline for your root account and one for your child account. With reference to the last post, these would be considered account-level pipelines.
+
+### What Are The Files
+
+|File|Description|
+|----|-----------|
+|child/aws_capability_iam.yml | Defines the DeveloperPolicy and DeveloperRole. DeveloperRole is the role that is assumed by your developers from the root account.|
+|root/aws_capability_iam.yml | Defines the ChildDeveloperPolicy and ChildDeveloperGroup.|
+|root/aws_capability_iam_users.yml | Defines the IAM User (in this case DeveloperPete) and assigns them to the ChildDeveloperGroup|
+
+## Taking It For A Spin
+
+If you're new to Inception Pipelines, head on over to the original post and set yourself up a pipeline. If not, copy over the files referenced above, add in the appropriate pipeline action and commit. Enjoy!
+
+## Wrapping Up
+
+And so, another month closes with another awesome Inception Pipeline post.
```

Comment:
Whilst true, you don't actually wrap up the post into a bite-sized conclusion or... wrap-up if you will